install-panel.sh

```diff
@@ -591,15 +591,28 @@ function firewall_firewalld {
   print_brake 70
 }
 
-function debian_based_letsencrypt {
-  # Install certbot and setup the certificate using the FQDN
-  apt install certbot -y
+function letsencrypt {
+  FAILED=false
+
+  # Install certbot
+  if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ]; then
+    apt-get install certbot -y
+  elif [ "$OS" == "centos" ]; then
+    [ "$OS_VER_MAJOR" == "7" ] && yum install certbot
+    [ "$OS_VER_MAJOR" == "8" ] && dnf install certbot
```

Comment:
Are you sure the `python2-certbot-nginx` and the `python3-certbot-nginx` are not required (7 and 8 respectively)?

Reply:
I think they are only used when using --webroot I think. When using --standalone it's not need because it creates a server itself. Either way we are stopping nginx :)

Reply:
Just tested without the `python3-certbot-nginx ` and `python2-certbot-nginx` and it works just fine. Then I guess it was correct I assumed it is only when you don't want to stop nginx.

Reply:
No, it's for the `certbot --nginx` which may come in handy if the user decides to obtain other certificates (since it works natively with Nginx in the same way certbot "natively" obtains certificate and performs verification with Apache). But this would require other changes in the script so I'll leave it for now, thanks for testing! :+1: